test/integration/certificates/issuing_controller_test.go

```diff
@@ -55,9 +56,14 @@ func TestIssuingController(t *testing.T) {
 		EnableOwnerRef: true,
 	}
 
+	ctx, cancel := context.WithTimeout(context.TODO(), time.Second*20)
+	defer cancel()
```

Comment:
Don't think this needs to move up here and given we pass `context.Background()` below it feels a bit confusing (we don't want a timeout on the context passed to NewController). Can you move this back to where it was/where it's used?